opentelemetry-api/src/opentelemetry/_logs/_internal/__init__.py

```diff
@@ -151,6 +151,9 @@ def get_logger(
             schema_url: Optional. Specifies the Schema URL of the emitted telemetry.
         """
 
+    def force_flush(self) -> None:
```

Comment:
For symmetry, what do you think about adding comments and making this an `@abstractmethod` like the one above?

Reply:
Makes sense, I changed it. I don't understand exactly what this method does, feel free to suggest a better docstring. 🙏🏻

Reply:
Sure, I'll take a look. Feel free to push your changes whenever you're ready.

Reply:
Oh, I pushed yesterday and it failed for some reason. 🤦🏻‍♂️ Pushed now again.

Reply:
Hmm, I've realized that my suggestion has spec compliance issues as force flush is [part of the SDK](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/sdk.md#forceflush) but is not indicated as part of the API. Will see if can get others to weigh in.

Reply:
👋 Hi there -- just wanted to let you know that we discussed this during the SIG meeting and unfortunately it looks like defining force_flush in the API, as I suggested, is problematic. Sorry for leading you astray on that!

I'm not quite sure what the right solution is yet as the API/SDK split is a little more complicated than I had initially thought, but I'm looking into it.

Reply:
Hi @iurisilvio -- after discussing with the SIG and looking into it myself, I think that your original solution is better than my suggestion (I also thought maybe we could create an interface in the SDK but that's probably over-thinking it). If you could revert we can move this forward. Thanks.

Reply:
Done! Reverted!